utils/tinystr/src/int_ops.rs

```diff
@@ -61,6 +61,28 @@ impl Aligned4 {
         (numeric & mask) == 0
     }
 
+    pub const fn is_ascii_lowercase(&self) -> bool {
```

Comment:
Note: The changes in this file are newly written. I added lots of tests, so I think they're correct, but it's worth a second look at the bit logic.

Reply:
could you add comments explaining where the numbers come from? The math does _look_ right to me too but comments would be useful

Reply:
Done